src/Standards/PSR12/Docs/ControlStructures/ControlStructureSpacingStandard.xml

```diff
@@ -0,0 +1,114 @@
+<documentation title="Control Structure Spacing">
+    <standard>
+    <![CDATA[
+    Control structures MUST have correct spacing.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: No space after the opening parenthesis.">
+        <![CDATA[
+if ($expr) {
+}
+        ]]>
+        </code>
+        <code title="Invalid: Space after the opening parenthesis.">
+        <![CDATA[
+if (<em> </em>$expr) {
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <code_comparison>
+        <code title="Valid: No space before the closing parenthesis.">
+        <![CDATA[
+if ($expr) {
+}
+        ]]>
+        </code>
+        <code title="Invalid: Space before the closing parenthesis.">
+        <![CDATA[
+if ($expr<em> </em>) {
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <code_comparison>
+        <code title="Valid: Each line in a multi-line control structure indented at least once. Default indentation is 4 spaces.">
+        <![CDATA[
+while (
+<em>    </em>$expr1
+<em>    </em>&& $expr2
+) {
+}
+        ]]>
+        </code>
+        <code title="Invalid: Some lines in a multi-line control structure not indented correctly.">
+        <![CDATA[
+while (
+<em></em>$expr1
+    && $expr2
+<em>  </em>&& $expr3
+) {
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <code_comparison>
```

Comment:
Suggestion: move this code sample block to be between the parentheses code samples and the indentation code sample.